qiskit/circuit/library/n_local/n_local.py

```diff
@@ -60,6 +60,70 @@ class NLocal(BlueprintCircuit):
 
     If specified, barriers can be inserted in between every block.
     If an initial state object is provided, it is added in front of the NLocal.
+
+    Examples:
+
+        .. jupyter-execute::
+
+            # Example 1:
+            #   "Blocks" are defined by `QuantumCircuit` objects.
+            #   The "basic rotation block" consists of a 1-depth 2-qubits `QuantumCircuit` object.
+            #   The "basic entanglement block" consists of a 3-depth 4-qubits `QuantumCircuit` object.
+            #   Every "rotation layer" consists of a vertical repetition over the basic rotation block.
+            #   Every "entanglement layer" consists of a `linear` repetition over
+            #   the basic entanglement block.
```

Comment:
I think you could move this explanation outside of the code block, then it would be rendered as normal text and be easier to read 🙂

Reply:
Sure